README.md

```diff
@@ -4666,12 +4665,12 @@ anonymous suggestions.
 
 ## <a name="ide"> Installing and Using IDEs </a>
 
-An *interactive development environment* (IDE) is a software tool that
+An *integrated development environment* (IDE) is a software tool that
```

Comment:
Did you mean "integrated"?

https://en.m.wikipedia.org/wiki/Integrated_development_environment